package.json

```diff
@@ -21,7 +21,7 @@
     "build:standalone": "cross-env NODE_ENV=production webpack --config webpack/standalone.babel.js",
     "preversion": "npm run lint && npm run test",
     "version": "auto-changelog -p && npm run build:dist && npm run build:standalone && git add CHANGELOG.md dist",
-    "prepublishOnly": "npm run build:lib && npm run build:lazy && npm run build:dist && node scripts/pre-publish.js && cp types/*.d.ts . && cp types/index.d.ts lazy",
+    "prepublishOnly": "npm run build:lib && npm run build:lazy && npm run build:dist && node scripts/pre-publish.js && cp types/*.d.ts . && cp types/lib/index.d.ts lib && cp types/lazy/index.d.ts lazy",
```

Comment:
Would this work?

```suggestion
    "prepublishOnly": "npm run build:lib && npm run build:lazy && npm run build:dist && node scripts/pre-publish.js && cp -r types/* .",
```

Reply:
I was thinking about that. I just tested it in my Linux machine and it works

Reply:
I was worried it might overwrite directories. But it does work. 👍🏽